packages/grid/src/elements/pane/Pane.spec.tsx

```diff
@@ -6,11 +6,24 @@
  */
 
 import React from 'react';
-import { render } from 'garden-test-utils';
+import { act, render } from 'garden-test-utils';
+import { Content } from './components/Content';
 import { Splitter } from './components/Splitter';
 import { Pane } from './Pane';
 import { PaneProvider } from './PaneProvider';
 
+// '@juggle/resize-observer' is the polyfill used by 'use-resize-observer'
+// we tap into the polyfilled ResizeObserver to trigger the useResizeObserver hook
+// this way we can test when the Pane is collapsed and ensure the Pane.Content is hidden
+jest.mock('@juggle/resize-observer', () => ({
+  ResizeObserver: function ResizeObserver(cb: () => void) {
+    const o = jest.requireActual('@juggle/resize-observer');
+
+    // @ts-expect-error resize observer callback is an artificial method added for testing
+    return new o.ResizeObserver((window.resizeObserverCallback = jest.fn(cb)));
```

Comment:
Setting the `resizeObserverCallback` to a property on the `window` allows us to capture the internal handler used by `useResizeObserver` and trigger updates to the hook with custom entries ([ResizeObserverEntry - MDN](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry))